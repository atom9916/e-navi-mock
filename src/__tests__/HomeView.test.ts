/**
 * @jest-environment jsdom
 */

import { render, screen } from '@testing-library/vue'
import HomeViewVue from '@/views/HomeView.vue'

describe('HomeViewビューのテスト', () => {
  test('画面が正しく表示される', () => {
    render(HomeViewVue)

    const h3 = screen.getByRole('heading')

    expect(h3).toHaveTextContent('ホーム')
  })
  test('スナップショットテスト', () => {
    const { container } = render(HomeViewVue)

    expect(container).toMatchSnapshot()
  })
})
