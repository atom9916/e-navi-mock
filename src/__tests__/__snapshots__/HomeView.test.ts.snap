// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeViewビューのテスト スナップショットテスト 1`] = `
<div>
  <main>
    <h3>
      ホーム
    </h3>
    <div>
      
      <div>
        <p>
           今日は 
          <span
            style="font-weight: 500; font-size: 1.4em; color: red;"
          />
           です 
        </p>
      </div>
      <div>
        <p>
           現在の時刻は 
          <span
            style="font-weight: 250; font-size: 1.4em;"
          />
           です 
        </p>
      </div>
      <div>
        <p>
          今日もお仕事、がんばりましょう…
        </p>
      </div>
      
    </div>
  </main>
</div>
`;
